app/lib/three_scale/bot_protection/controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module ThreeScale
+  module BotProtection
+    module Controller
+      include Recaptcha::Adapters::ControllerMethods
+
+      private
+
+      def bot_check(options = { flash: true })
+        return true if logged_in?
```

Comment:
I took this behavior from the previous code. I didn't question the idea of disabling bot protection for logged users, but I'm open to remove that restriction. I think it's OK to remove it since the three screens where the captcha will be shown are only available for unsigned clients (password recover, signup, and soon: login), and, maybe in the future we want to protect other screens inside the developer portal, e.g. `/admin/messages/new`

WDYT?

CC: @mayorova

Reply:
I think it makes sense to remove that restriction then.

Reply:
Done in https://github.com/3scale/porta/pull/3645/commits/ef5fabfd2a8ff5f5f9aab41fded7c18cb07e9d16